rfcs/000-support-rdb-models.md

```diff
@@ -0,0 +1,122 @@
+- Feature Name: Support Relational Database Models
```

Comment:
is it correct to have this submission under this PR ?

Reply:
Oops, something went wrong!